Sources/Plasma/PubUtilLib/plAudio/plWin32Sound.cpp

```diff
@@ -106,11 +106,14 @@ void plWin32Sound::Update()
     if (buf != nullptr) {
         plSrtFileReader* srtReader = buf->GetSrtReader();
         if (srtReader != nullptr) {
-            while (plSrtEntry* nextEntry = srtReader->GetNextEntryStartingBeforeTime((uint32_t)(GetActualTimeSec() * 1000.0f))) {
-                if (plgAudioSys::AreSubtitlesEnabled()) {
-                    // add a plSubtitleMsg to go... to whoever is listening (probably the KI)
-                    plSubtitleMsg* msg = new plSubtitleMsg(nextEntry->GetSubtitleText(), nextEntry->GetSpeakerName());
-                    msg->Send();
+            uint32_t currentTimeMs = (uint32_t)(GetActualTimeSec() * 1000.0f);
+            if (currentTimeMs <= srtReader->GetLastEntryEndTime()) {
+                while (plSrtEntry* nextEntry = srtReader->GetNextEntryStartingBeforeTime((uint32_t)(GetActualTimeSec() * 1000.0f))) {
```

Comment:
Is `(uint32_t)(GetActualTimeSec() * 1000.0f)` the same value as `currentTime` here?

Reply:
Well dangit, I think I borked that in the merge. Don't do merges late at night, kids. Yes, the `currentTime` variable was supposed to be used in both places. I'll fix.